src/scss/skin-modern/components/_errormessageoverlay.scss

```diff
@@ -18,11 +18,12 @@
     right: 3em;
     text-align: center;
     user-select: text;
+    white-space: pre-line; // enable linebreak in text
 
-    // Vertically center the label (not exactly, but works for single line errors)
+    // Vertically center the label
     & {
-      margin-top: -1em;
       top: 50%;
+      transform: translateY(-50%);
```

Comment:
Does this work in IE9 & IE11?

Reply:
afaik yes but i will test it in a sec.

Reply:
It is working find in IE11. Unfortunately i'm currently not able to test it in IE9 because the player v8 is currently not working there.

Reply:
You can test the CSS transform by itself.